src/brs/fluxcapacitor/FluxCapacitorImpl.java

```diff
@@ -6,14 +6,15 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 
 public class FluxCapacitorImpl implements FluxCapacitor {
 
     private final PropertyService propertyService;
     private final Blockchain blockchain;
 
     // Map of Flux Value -> Change Height -> Index of ValueChange in FluxValue. Used as a cache.
-    private final Map<FluxValue<?>, Map<Integer, Integer>> valueChangesPerFluxValue = new HashMap<>();
+    private final Map<FluxValue<?>, Map<Integer, Integer>> valueChangesPerFluxValue = new ConcurrentHashMap<>();
```

Comment:
These are the kinda gotchas I mentioned caused by the changes in the runtime

Reply:
I've found this one and that other for DB stuff with the *lambda* expression being rejected on run-time.